src/utils/MessageBuilder.ts

```diff
@@ -213,4 +232,104 @@ export default class MessageBuilder {
       embeds: [embed],
     };
   }
+
+  static blueChooseMessage(ballChasers: ReadonlyArray<PlayerInQueue>): MessageOptions {
```

Comment:
There's a lot of duplication between the blue and orange choose messages. Do you plan on cleaning this up in your next PR?